controller/src/beerocks/master/db/db.cpp

```diff
@@ -19,6 +20,66 @@ using namespace beerocks_message;
 using namespace son;
 using namespace net;
 
+static const std::string timestamp_str            = "timestamp";
+static const std::string timelife_delay_str       = "timelife";
+static const std::string initial_radio_enable_str = "initial_radio_enable";
+static const std::string initial_radio_str        = "initial_radio";
+static const std::string selected_band_enable_str = "selected_band_enable";
+static const std::string selected_bands_str       = "selected_bands";
+
+// static
+std::string db::type_to_string(beerocks::eType type)
+{
+    if (type == beerocks::eType::TYPE_GW) {
+        return std::string("gateway");
+    } else if (type == beerocks::eType::TYPE_IRE) {
+        return std::string("ire");
+    } else if (type == beerocks::eType::TYPE_IRE_BACKHAUL) {
+        return std::string("ire_bh");
+    } else if (type == beerocks::eType::TYPE_SLAVE) {
+        return std::string("slave");
+    } else if (type == beerocks::eType::TYPE_CLIENT) {
+        return std::string("client");
+    } else if (type == beerocks::eType::TYPE_ETH_SWITCH) {
+        return std::string("eth_switch");
+    } else if (type == beerocks::eType::TYPE_ANY) {
+        return std::string("any");
+    } else if (type == beerocks::eType::TYPE_UNDEFINED) {
+        return std::string();
+    }
+    return std::string();
+}
+
+std::string db::client_db_entry_from_mac(const sMacAddr &mac)
+{
+    std::string db_entry = tlvf::mac_to_string(mac);
+    std::replace(db_entry.begin(), db_entry.end(), ':', '_');
```

Comment:
Why do we need to replace ':' character?

Reply:
uci doesn't support `:` in the section name